mods/default/furnace.lua

```diff
@@ -3,8 +3,10 @@
 -- Formspecs
 --
 
+local update_step = tonumber(minetest.setting_get("default_furnace_update_step")) or 0.1
```

Comment:
As the number is in seconds, not serversteps, a name like "update_time" might be less misleading.